docs/src/faq.md

```diff
@@ -11,21 +17,44 @@ covariance between estimated coefficient parameters.  For example, an estimated
 polynomial can be represented as a multivariate Normal distribution,
 with a variance-covariance matrix.  To use this, define the parameter
 in the component with a vector type, like here:
```

Comment:
@davidanthoff should I remove this FAQ question all together since we no longer support this?

Reply:
Yes, I think the docs should always match what we ship!